modules/yasmin-hardware.nix

```diff
@@ -13,6 +13,7 @@
   boot.initrd.kernelModules = [ ];
   boot.kernelModules = [ ];
   boot.extraModulePackages = [ ];
+  boot.kernelPatches = [ { name = "uprobes"; patch = null; extraConfig = "CONFIG_UPROBE_EVENTS y"; } ];
```

Comment:
Please move this a uprobes.nix module and import this in yasmin.nix.
This file is auto-generated mostly.

Reply:
done.